src/ZendeskApi_v2/Models/Users/GroupUserExportResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using ZendeskApi_v2.Models.Organizations;
+
+namespace ZendeskApi_v2.Models.Users
+{
+    public class GroupUserExportResponse
+    {
+        [JsonProperty("count")]
+        public long Count { get; set; }
+
+        [JsonProperty("end_time")]
+        [JsonConverter(typeof(UnixDateTimeConverter))]
+        public DateTime EndTime { get; set; }
```

Comment:
let's make this a DattimeOffset

Reply:
Then it should also change in GroupTicketExportResponse (The class is based on that)... but that would be a breaking change so do not know if what is best. Make it right here and not make it consistent across the framework or have it same format. Up to you